nightfall-client/src/routes/transform.mjs

```diff
@@ -0,0 +1,20 @@
+import express from 'express';
+import logger from '@polygon-nightfall/common-files/utils/logger.mjs';
+import transform from '../services/transform.mjs';
+
+const router = express.Router();
+
+router.post('/', async (req, res, next) => {
+  try {
+    const { rawTransaction: txDataToSign, transaction } = await transform(req.body);
+    logger.debug('returning raw transaction');
```

Comment:
You don't need these log instructions anymore, on the routes the logging of the payload/response is carried out automatically.